sync-wp/components/config/screen.py

```
@@ -0,0 +1,105 @@
+        # Main layout
+        main_layout = QVBoxLayout(self)
+        
+        # Auth group box
+        auth_group = QGroupBox("WordPress Authentication")
+        auth_layout = QFormLayout()
+        
+        self.username_input = QLineEdit()
+        self.password_input = QLineEdit()
+
+        
+        auth_layout.addRow("Username:", self.username_input)
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)